lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -255,11 +262,12 @@ sub filter_loh {
 #Get rid of intermediate files
 sub cleanup {
     my $self = shift;
+    my $outdir = shift;
     my @patterns = qw(*.Somatic* *.readcounts *.hc *.LOH
```

Comment:
Right I see now that these are the files being removed and not kept.  Duh.  This makes sense!